Don't use git(1) to determine the current version. Also don't embed
date(1) into the binary to make the build reproducible.

--- gosec-2.30.0.orig/Makefile
+++ gosec-2.30.0/Makefile
@@ -18,9 +18,9 @@
 GOVULN_MIN_VERSION = 17
 GO_VERSION = 1.27
 LDFLAGS = -ldflags "\
-	-X 'main.Version=$(shell git describe --tags --always)' \
-	-X 'main.GitTag=$(shell git describe --tags --abbrev=0)' \
-	-X 'main.BuildDate=$(shell date -u +%Y-%m-%dT%H:%M:%SZ)'"
+	-X 'main.Version=$(GIT_TAG)' \
+	-X 'main.GitTag=$(GIT_TAG)' \
+	-X 'main.BuildDate=$(BUILD_DATE)'"
 
 default:
 	$(MAKE) build
